main.py

```diff
@@ -0,0 +1,6 @@
+urls = []
+for path in paths:
+  url = 'https://'
+  url += domain
+  url += path
+  urls.append(url)
```

Comment:
let's simplifying it with a list comprehension like so:
```suggestion
urls = ['https://' + domain + path for path in paths]
```